build.gradle

```diff
@@ -309,6 +309,16 @@ subprojects {
     }
 }
 
+task('generate-docker') {
+    dependsOn(':airbyte-workers:build')
```

Comment:
i think you can depend on `:airbyte-workers:buildDockerImage-${artifact name}` here instead. it might make our lives easier in the future to if the names don't have the artifact name suffix at the end, but doesn't have to be a concern now.

Reply:
I am not sure if that's possible. I am trying to do it now and I am running into many issue mainly because the task are generated on the fly.

Reply:
@cgardens your main goal is to have it not require tests to build, right?

If so, we could use `assemble` instead of build?

Reply:
Done